pyexchange/ddex.py

```diff
@@ -163,6 +207,56 @@ def cancel_order(self, order_id: str) -> bool:
 
         return success == 0
 
+    def get_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
+        assert(isinstance(pair, str))
+        assert(isinstance(page_number, int))
+
+        per_page = 100
+        page_filter = f"page={page_number}&per_page={per_page}"
+        result = self._http_get_signed(f"/v2/markets/{pair}/trades/mine?{page_filter}", {})['data']
+        totalPages = result['totalPages']
+        currentPage = result['currentPage']
+        self.logger.debug(f'totalPages={totalPages};currentPage={currentPage}')
+
+        # Oldest trades are on first page
```

Comment:
oldest or newest...?

Reply:
newest are now on top since the breakage, was an old comment will update !